src/windows/symbol.rs

```diff
@@ -509,26 +509,34 @@ pub(super) fn append_dummy_symbol(mut syms: PDBSymbols, name: &str) -> PDBSymbol
 pub(super) fn symbolic_to_pdb_symbols(
     syms: PeSymbolIterator,
     exception_data: Option<&ExceptionData<'_>>,
-    name: &str,
+    module_name: &str,
 ) -> PDBSymbols {
     let mut pdb_syms = PDBSymbols::default();
-    let sizes = match exception_data {
-        Some(data) => {
-            let mut sizes = HashMap::new();
-            for function in data {
-                if let Ok(function) = function {
-                    if function.end_address <= function.begin_address {
-                        continue;
-                    }
-                    sizes.insert(
-                        function.begin_address,
-                        function.end_address - function.begin_address,
-                    );
-                }
+
+    let module_name = if module_name.is_empty() {
+        String::from("<unknown>")
+    } else {
+        format!("<unknown in {}>", module_name)
+    };
+
+    if let Some(data) = exception_data {
+        for element in data {
+            if let Ok(function) = element {
+                pdb_syms.insert(
+                    function.begin_address,
+                    PDBSymbol {
+                        name: module_name.clone(),
+                        is_public: false,
+                        is_multiple: false,
+                        rva: function.begin_address,
+                        len: function.end_address - function.begin_address,
```

Comment:
skip functions where `end < begin`, I don't know if it can happen in real life but who knows

Reply:
Let's skip them, we've already seen corrupted/buggy debug info so better be safe than sorry